supplier_app/tests/factory_boy.py

```diff
@@ -49,6 +49,7 @@ class Meta:
     cuit = "1234569"
     comments = "Justificacion falsa"
     payment_type = "Banco"
```

Comment:
`payment_type` should be one of a list of possible payment type values

Reply:
resolved